tests/suites/test_suite_x509parse.function

```diff
@@ -818,6 +818,42 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C */
+void mbedtls_x509_get_name( char * rdn_sequence, int exp_ret )
```

Comment:
```suggestion
void mbedtls_x509_get_name( char *rdn_sequence, int exp_ret )
```

Reply:
The space following the `*` is consistent with the style of other functions in this file (e.g. on [this](https://github.com/Mbed-TLS/mbedtls/pull/6391/files#r999668812) line). Should I change it anyway?

Reply:
We're going to normalize horizontal whitespace soon. Let's not bother with minor deviations, especially when the surroundings are already inconsistent.

Reply:
agreed, TLS 1.3 implementation reflex, sorry for the noise.